infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/UnitTestClusterManager.java

```diff
@@ -0,0 +1,125 @@
+package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
+
+import com.google.common.collect.ImmutableList;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.logreplication.infrastructure.ClusterDescriptor;
+import org.corfudb.infrastructure.logreplication.infrastructure.NodeDescriptor;
+import org.corfudb.infrastructure.logreplication.infrastructure.TopologyDescriptor;
+import org.corfudb.infrastructure.logreplication.proto.LogReplicationClusterInfo.ClusterRole;
+import org.corfudb.infrastructure.logreplication.proto.LogReplicationClusterInfo.TopologyConfigurationMsg;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+import static org.corfudb.infrastructure.logreplication.proto.LogReplicationClusterInfo.ClusterRole.ACTIVE;
+import static org.corfudb.infrastructure.logreplication.proto.LogReplicationClusterInfo.ClusterRole.STANDBY;
+
+@Slf4j
+public class UnitTestClusterManager extends CorfuReplicationClusterManagerBaseAdapter {
```

Comment:
How different is this from FileReaderClusterManager? Can't that one be used with a fixed file?

Reply:
Removed that one. It was not used.